agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -508,6 +523,8 @@ private static void applyRpcClientSpan(
       target = rpcSystem;
     }
     telemetryBuilder.setTarget(target);
+    // RPC client/server metrics only has duration
+    telemetryBuilder.addProperty(MS_PROCESSED_BY_METRIC_EXTRACTORS, "True");
```

Comment:
it's not clear to me why this doesn't need `isPreAggregated` conditional?

Reply:
this is not needed. good catch. it has been checked under exportRemoteDependency and exportRequest.  old code before i added the isPreAggregated attribute.